phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -2650,6 +2685,8 @@ else if (!SchemaUtil.isSystemTable(Bytes.toBytes(SchemaUtil.getTableName(schemaN
                         .setIndexes(Collections.<PTable>emptyList())
                         .setPhysicalNames(ImmutableList.<PName>of())
                         .setColumns(columns.values())
+                        .setViewTTL(0L)
```

Comment:
Why are we setting it to 0L? Shouldn't we set it to `viewTTL` extracted above?
Also, can we extract `0L` as a special constant value with a comment on what it indicates (never expire?)

Reply:
My understanding is this is special handling for SYSTEM table as the comment above says. We do not want to set TTL for any SYSTEM tables. 
But your comment about extracting 0L as a special constant makes sense. I will do that.